### ISSUE TITLE
Create a Your Reviews page 

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeCENMigration#1326]
Per FreeREG, add FreeCEN format of form

**Comment by DeniseColbert** [comment]
Check FreeREG journey works too.

**Comment by DeniseColbert** [comment]
See #250 and #1454

**Comment by AlOneill** [comment]
[DRAFT Your reviews](https://test3.freecen.org.uk/cms/about/draft-your-reviews) page created. Needs work … and content …

**Comment by DeniseColbert** [comment]
@DeniseColbert to add content

**Comment by DeniseColbert** [comment]
@PatReynoldsFUG to send some nice feedback to me from Contacts

**Comment by AlOneill** [comment]
Added two dummy reviews with info re markup, etc. Ideally this would be done as an HTML comment but Refinery deletes them (which is understandable)

**Comment by PatReynoldsFUG** [comment]
Unfortunately I now don't have access to the archived specific-county contacts.

**Comment by DeniseColbert** [comment]
@DeniseColbert to check donation feedback

**Comment by DeniseColbert** [comment]
@PatReynoldsFUG to liaise with Chris M to place a call to Coordinators in the Newsletter to ask for any positive feedback they have on FreeCEN/their volunteers etc.

**Comment by PatReynoldsFUG** [comment]
Wrote to Chris 3 Apr 2024

**Comment by PatReynoldsFUG** [comment]
Wrote to Chris again 25/7

**Comment by PatReynoldsFUG** [comment]
Asked Wikitreers 20/8/2024

**Comment by PatReynoldsFUG** [comment]
Asked Wikitreers 20/8/2024

**Comment by DeniseColbert** [comment]
@PatReynoldsFUG to check any feedback from the newsletter

**Comment by DeniseColbert** [comment]
@PatReynoldsFUG now has a few nice testimonials, to look on FreeCEN FB group and Denise will ask Helen to post in Ancestry UK asking for them too.

**Comment by DeniseColbert** [comment]
Pat to search FB for FreeCEN and see if anything useful comes back.

**Comment by PatReynoldsFUG** [comment]
Added another review - which was all I could find.

**Comment by PatReynoldsFUG** [comment]
Nothing else I could find on FB or more widely.

**Comment by DeniseColbert** [comment]
Waiting to hear back from Helen

**Comment by DeniseColbert** [comment]
Pat has replied to Helen's post: Helen to monitor

**Comment by DeniseColbert** [comment]
@PatReynoldsFUG to ask Helen if anything cam back from this.

**Comment by PatReynoldsFUG** [comment]
I have found one quotable  comment on Helen's group, and have started a page to collect from there and other places: https://docs.google.com/document/d/1xtE9UyCfCjONuk70h6s8DLEkctIDJs2qbLnTAuYUduk/edit?usp=sharing (accessible when logged in with a FUG email address if I've got the privacy setting correct.

Updated: 
Think we have enough now to go ahead.

**Comment by DeniseColbert** [comment]
@AlOneill This story is about emulating the Reviews page on FreeREG over on FreeCEN. Pat has collected a few nice things people have said about it on her doc above. Do you think you might be able to create the page, or would a Ruby dev need to do it now we're not using Refinery?

**Comment by AlOneill** [comment]
@DeniseColbert 
I can probably create it (having updated the Newsletters page successfully, with Vino's help). I'll ask for further help if necessary.

**Comment by AlOneill** [comment]
@Vino-S 
Updating this page on GitHub is a little different from updating the Newsletters page. The changes are more complicated: if a form is required, then that needs adding before publication on live (I don't do Google forms); other people will need to review the page and sign it off, presumable on test. So, how does that affect the workflow? Do I commit the updated DRAFT page to Master? Or do I do something different with it?

Many thanks!

**Comment by AlOneill** [comment]
Draft page updated in GitHub. Awaiting publishing on test.

It will need a Google form for review submissions. This is not my area of expertise.

**Comment by DeniseColbert** [comment]
I will find the embed code for the right form and send to Vino for inclusion: thanks @AlOneill !

**Comment by DeniseColbert** [comment]
To be linked from the About page below sidebar (in See more box).

"See more" should be changed to "More about FreeCEN"

**Comment by AlOneill** [comment]
@DeniseColbert 
The menu on the About page headed "See also" contains the link to "FreeCEN Newsletters".  Could we not use that for the link to "Your reviews"? There is currently no "See more" menu box on CEN > About, unlike REG > About.

**Comment by DeniseColbert** [comment]
Sorry @AlOneill, I meant "See also" rather than "See more". That is where I am proposing the link to Your Reviews should go, along with the FreeCEN Newsletters.

They should all be: "More about [Project]" for consistency.

**Comment by AlOneill** [comment]
@DeniseColbert 
Are any other changes required? If not and you are happy about how the form works, I'll tidy up the "Your reviews" page, including the removal of "DRAFT" from its name and then create a link from the "About" page – adding it to the "More about…" page menu. 

Should all the "More about…" menus be placed 'at the foot of the sidebar' or can we place them after the "On this page" menus, where the current menu on FreeCEN > About is located? I am concerned that FreeCEN users will be confused if the link to the Newsletters page suddenly moves to near the foot of the About page.

**Comment by DeniseColbert** [comment]
I can't see the page @AlOneill can you provide a link to it?

My preference is for nothing to be at the foot of the sidebar, I think it should all be directly below the 'On this page' section.

**Comment by AlOneill** [comment]
@DeniseColbert 
The draft of the [Your reviews page](https://test.freecen.org.uk/about/draft-your-reviews) is on test.

The only small snag with placing the "More about …" menus directly under the "On this page" menus is to do with source order: mobile users and screen reader users (if reading the page sequentially) will encounter both before getting into the main contents of the page. However, I agree that this option is overall better than placing each at the foot of the sidebar (which is where I was asked to place the FreeREG one originally, now several years ago).

**Comment by DeniseColbert** [comment]
Thanks @AlOneill 

A couple of queries: should there be a space in the first quote on the snip below?

Are the second and fourth quotes effectively the same? First word of each notwithstanding.

<img width="883" height="763" alt="Image" src="https://github.com/user-attachments/assets/4ae0e106-b840-415d-840e-7f023139508d" />

I looked for some more and have sent you a doc to see if you like any to include @AlOneill

**Comment by AlOneill** [comment]
@DeniseColbert Yes, there should be a space, but I understood that I was not to correct these reviews in any way. Are you asking me to add a space? And yes, those reviews are the same – good catch – I'll fix that.

I'll take a look at those additional reviews – thanks.

**Comment by DeniseColbert** [comment]
@AlOneill thanks! I think the lack of a space is Pat's error in creating the draft document text, rather than a typo in the actual quote. Fine to fix that.

**Comment by stephencoward-source** [comment]
Still waiting for reply from Alison. Denise to follow up.

**Comment by AlOneill** [comment]
@DeniseColbert @stephencoward-source 
I'll get to this when I have the energy and the Aug Newsletter has been dealt with. And there is no heat wave. Apologies for the delay.

**Comment by AlOneill** [comment]
@Vino-S 
I'm ready to update the draft page with the corrections and additional reviews. Where do I find the current version of the page that is on test? Looking at my original branch (AlOneill-patch-1), it is missing the form, so I do not know what to do. Many thanks!
